zou/cli.py

```diff
@@ -136,26 +136,30 @@ def create_admin(email, password):
     Create an admin user to allow usage of the API when database is empty.
     """
     try:
-        auth.validate_password(password)
```

Comment:
Why did you removed the password validation?

Reply:
Ah, that is a mistake from applying the patch file, my bad.

I fixed it.